source/renderer/app/components/staking/stake-pools/StakePoolsSearch.tsx

```diff
@@ -96,7 +101,13 @@ function StakePoolsSearchComponent({
       {({ scrollElementRef }) => (
         <div className={styles.component}>
           <div className={styles.container}>
-            <SVGInline svg={searchIcon} className={styles.searchIcon} />
+            <SVGInline
+              svg={searchIcon}
+              className={classnames(
```

Comment:
Would make sense to refactor this to use array like? I think this is the pattern used across the code base 

```
    const checkboxStyles = classnames([
      styles.checkbox,
      isCheckboxDisabled ? styles.disabled : null,
    ]);
```

Reply:
Depends. What are the benefits of using the array like syntax for in this context?

Reply:
Personally I prefer Shawn's approach since the code looks almost identical without brackets so they give no value IMO, but I agree we should have a pattern established. I suggest rising this on a Dev Meeting.